AzureCalling/Controllers/LobbyViewController.swift

```diff
@@ -115,6 +116,12 @@ class LobbyViewController: UIViewController, UITextFieldDelegate {
         previewCenterImageView.isHidden = true
     }
 
+    private func openAudioDeviceDrawer() {
+        let audioDeviceSelectionViewController = AudioDeviceSelectionViewController()
+        audioDeviceSelectionViewController.modalPresentationStyle = .overCurrentContext
```

Comment:
Could `audioDeviceSelectionViewController.modalPresentationStyle = .overCurrentContext` remove to the `SelectionViewController`?